conda_smithy/feedstock_content/build-locally.py

```diff
@@ -16,7 +16,14 @@ def setup_environment(ns):
 
 def run_docker_build(ns):
     script = ".scripts/run_docker_build.sh"
-    subprocess.check_call([script])
+    if ns.debug:
+        env = os.environ.copy()
```

Comment:
I think it is a little cleaner to move this line above the if-statement and drop the else clause

Reply:
I think this should be moved to the `setup_environment` function

Reply:
Yeah that would be fine too

Reply:
Done 👍